.github/workflows/build-executor.yml

```diff
@@ -26,6 +27,26 @@ jobs:
           WORKING_DIR: ${{ inputs.working_dir }}
           JAVA_HOME: /usr/lib/jvm/default-jvm   
 
+      - name: Run ballerina build for dev
+        if: inputs.bal_central_environment == 'DEV'
+        uses: ballerina-platform/ballerina-action@2201.7.0
+        with:
+          args:
+            pack
+        env:
```

Comment:
We need to add the relevant tokens as well - https://github.com/ballerina-platform/module-ballerinax-health.hl7v2/blob/main/.github/workflows/build-executor.yml#L38-L39

Please add for both stage and dev

Reply:
Done